cleanrl/atari/v_c51_atari.py

```diff
@@ -0,0 +1,346 @@
+    parser.add_argument('--capture-video', type=lambda x:bool(strtobool(x)), default=False, nargs='?', const=True,
+                        help='weather to capture videos of the agent performances (check out `videos` folder)')
+    parser.add_argument('--wandb-project-name', type=str, default="cleanRL",
+                        help="the wandb's project name")
+    parser.add_argument('--wandb-entity', type=str, default=None,
+                        help="the entity (team) of wandb's project")
+    
+    # Algorithm specific arguments
+    parser.add_argument('--num-envs', type=int, default=8,
+        help='the number of parallel game environments')
+    parser.add_argument('--num-steps', type=int, default=128,
+        help='the number of steps to run in each environment per policy rollout')
+    parser.add_argument('--num-minibatches', type=int, default=4,
+        help='the number of mini-batches')
+    parser.add_argument('--update-epochs', type=int, default=16,
+        help="the K epochs to update the policy")
+    parser.add_argument('--n-atoms', type=int, default=51,
+                        help="the number of atoms")
+    parser.add_argument('--v-min', type=float, default=-10,
+                        help="the number of atoms")
+    parser.add_argument('--v-max', type=float, default=10,
+                        help="the number of atoms")
+    parser.add_argument('--gamma', type=float, default=0.99,
+                        help='the discount factor gamma')
+    parser.add_argument('--target-network-frequency', type=int, default=1000,
+                        help="the timesteps it takes to update the target network")
+    parser.add_argument('--max-grad-norm', type=float, default=0.5,
+                        help='the maximum norm for the gradient clipping')
+    parser.add_argument('--batch-size', type=int, default=32,
+                        help="the batch size of sample from the reply memory")
+    parser.add_argument('--start-e', type=float, default=1.,
+                        help="the starting epsilon for exploration")
+    parser.add_argument('--end-e', type=float, default=0.01,
+                        help="the ending epsilon for exploration")
+    parser.add_argument('--exploration-fraction', type=float, default=0.10,
+                        help="the fraction of `total-timesteps` it takes from start-e to go end-e")
+    parser.add_argument('--learning-starts', type=int, default=80000,
+                        help="timestep to start learning")
+    parser.add_argument('--train-frequency', type=int, default=4,
+                        help="the frequency of training")
+    args = parser.parse_args()
+    args.batch_size = int(args.num_envs * args.num_steps)
+    args.minibatch_size = int(args.batch_size // args.num_minibatches)
+    if not args.seed:
+        args.seed = int(time.time())
+    return args
+
+# TRY NOT TO MODIFY: setup the environment
```

Comment:
use a `if __name__ == "__main__": here and indent the code below

Reply:
Updated